api/projects/views.py

```diff
@@ -97,12 +93,10 @@ def get_queryset(self):
 
     def perform_create(self, serializer):
         project = serializer.save()
-        if self.request.user.is_anonymous:
-            return
-
-        UserProjectPermission.objects.create(
-            user=self.request.user, project=project, admin=True
-        )
+        if isinstance(self.request.user, FFAdminUser):
+            UserProjectPermission.objects.create(
+                user=self.request.user, project=project, admin=True
+            )
```

Comment:
Interesting, this is going to mean that the key which creates the project will not have access to the project once its created it, right? (unless it's an admin key) Perhaps this is what you'd expect but it certainly feels slightly odd. 

I guess, to be fair, people are unlikely to be creating projects via these API keys anyway.

Reply:
Yeah, this is an interesting one, we can't really give it admin permission like we give it to the user because that will mean updating the assigned role

Reply:
Yeah, I tend to agree.